cli/crates/server/src/proxy.rs

```diff
@@ -120,11 +126,15 @@ async fn graphql_inner(
     mut req: Request<Body>,
     path: &str,
 ) -> Result<impl IntoResponse, impl IntoResponse> {
+    const BODY_LIMIT: usize = 1_024 * 1_024 * 512;
```

Comment:
What's this based on?

Reply:
An arbitrary number, really.

Reply:
Does CF have a max body size? Would probably be more accurate to use that

Reply:
Cloudflare Plan	Maximum body size
Free	100 MB
Pro	100 MB
Business	200 MB
Enterprise	500 MB (by default)

Reply:
We could pin it to whatever we're using since it'd mimic production (and possibly add a flag to make it larger for on-prem)

Reply:
Although looks like it's already around the largest size so can be left as is I guess

Reply:
@yoav-lavi hah, that's funny, I wasn't following Cloudflare there. I'll add a comment referring to the numbers you quoted. Thanks so much for investigating!